index.js

```diff
@@ -20,6 +20,8 @@ var logger = require('./services/logger');
 var Integrator = require('./integrations');
 var errorHandler = require('./services/error-handler');
 var ApimapSender = require('./services/apimap-sender');
+var ipWhitelist = require('./services/ip-whitelist');
+var filterIp = require('./middlewares/ip-whitelist');
```

Comment:
ipAuthorizer / ipValidator / ipChecker seems to be better middleware name options.